linkerd/core/src/main/scala/io/buoyant/linkerd/telemeter/UsageDataTelemeter.scala

```diff
@@ -206,20 +207,25 @@ case class UsageDataTelemeter(
     Handler("/admin/metrics/usage", new UsageDataHandler(metricsService, config, pid, orgId, metrics))
   )
 
+  def jitter(i: Duration): Duration = (Random.nextGaussian() * i.inSeconds).toInt.seconds
+
   // Only run at most once.
   def run(): Closable with Awaitable[Unit] =
     if (started.compareAndSet(false, true)) run0()
     else Telemeter.nopRun
 
   private[this] def run0() = {
-    val _ = client(metrics)
+    val initialCall = DefaultTimer.twitter.schedule(jitter(1.minute)) {
+      val _ = client(metrics)
+    }
 
-    val task = DefaultTimer.twitter.schedule(DefaultPeriod) {
+    val recurringCall = DefaultTimer.twitter.schedule(DefaultPeriod.plus(jitter(1.minute))) {
       val _ = client(metrics)
     }
```

Comment:
Doesn't this initiate _two_ request loops?  i.e. `initialCall` is a task that will continue to submit updates every ~minute?

Should the first one just be `Future.sleep(jitter(1.minute).before(client(metrics))` ?